File: app/src/main/java/dev/trindade/robokide/MainActivity.java
package dev.trindade.robokide;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import robok.trindade.methods.*;

import dev.trindade.robokide.ui.fragments.EditorFragment;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle _savedInstanceState) {
    super.onCreate(_savedInstanceState);
    setContentView(R.layout.activity_main);
	EditorFragment fragment = new EditorFragment();
	goToFragment(fragment);
  }
  
  public void goToFragment(Fragment frag){
	  FragmentManager fragmentManager = getSupportFragmentManager();
	  FragmentTransaction transaction = fragmentManager.beginTransaction();
	  transaction.replace(R.id.fragment_container, frag);
	  transaction.addToBackStack(null);
	  transaction.commit();
  }
  
}

File: app/src/main/java/dev/trindadeaquiles/robokide/ui/activities/MainActivity.kt
package dev.trindadeaquiles.robokide.ui.activities

import android.os.Bundle

import com.google.android.material.transition.MaterialSharedAxis

import dev.trindadeaquiles.robokide.R
import dev.trindadeaquiles.robokide.ui.fragments.home.HomeFragment
import dev.trindadeaquiles.robokide.ui.base.RobokActivity

class MainActivity : RobokActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        if (savedInstanceState == null) {
            openFragment(HomeFragment(MaterialSharedAxis.X))
        }
        
    }
}